content/2016-12-26_komunitni-python-stanek-na-linuxdays.md

```diff
@@ -0,0 +1,90 @@
+Title: Komunitní Python stánek na LinuxDays
+Date: 2016-12-26 13:28:00
+Modified: 2016-12-26 13:28:00
+Author: Veronika Gabrielová
+Twitter: lspdv
+Gravatar: gabrielova.veronika@gmail.com
+About: Veronička je velmi aktivní v české [Python](http://python.cz/) komunitě. Motá se hodně kolem [PyLadies](http://pyladies.cz/), účastnicím po kurzech pomáhá najít vysněnou práci a právě buduje rozjezd i dalších navazovacích kurzů a srazů pro všechny zájemce o Python, kteří jsou v začátcích. Také již pracuje jako programátorka. Nemá ráda datle. Má ráda vše ostatní kromě datlí. Nejvíc Žufánka a Pho. Nejčastěji ji potkáš v Praze na Pyvo srazech či na PyWorkingu a PyLadies.
+
+### Organizátoři LinuxDays nás oslovili, zdali chceme využít možnosti prezentovat se na známé akci LinuxDays. 
+
+```
```

Comment:
V pořádku, já jsem nevěděl, co byl přesně záměr. S MD se nedají dělat grafické úpravy. Je to na to, aby se významově (významy jako "kód", "citace", "důležité", "méně důležité", ...) označil text a potom ho mohlo něco (třeba ten program, který tvoří blog) pochopit a nějak zobrazit. Nelze s ním ovlivnit, jak budou věci přímo vypadat - tzn. jak si který význam vyloží který program. Jak můžeš vidět, GitHub zobrazí blok kódu se šedým pozadím, blog ho zobrazí s černým a snaží se zvýrazňovat syntaxi. Přes MD nelze říct "tady chci červené pozadí" (a je to tak dobře).

Ty bloky začínající a končící třemi "zpětnými apostrofy" (nebo jak se to jmenuje? anglicky backtick) jsou určené pro to, když chceš do článku vložit kód programovacího jazyka. Dokonce lze napovědět, o který se jedná:

    ```python
    import antigravity
    ```

Program, který si toto v MD přečte, může potom hezky zvýraznit obsah toho bloku, jako by to byl Python. Fungují i další jazyky.

A je dobře, že se tu o tom bavíme - oba se něco naučíme a já můžu pak napsat lepší dokumentaci k tomu blogu :)